### ISSUE TITLE
[UX] Add "You're checked in." Screenshot to Poster

### ISSUE THREAD

**Issue by loripusey** [department-of-veterans-affairs/va.gov-team#60133]
## Description
We have been asked by IVC stakeholders to add a screenshot of the "You're checked in" page to the poster so that Veterans will know what the end-point is and that they have been checked in.

[Slack thread](https://dsva.slack.com/archives/C022AC2STBM/p1686582407883509)

## Tasks
- [ ] Update the poster to include a screenshot of the "You're checked in" page

## AC 
- [ ] Poster has been updated

**Comment by loripusey** [comment]
This is under the epic to re-design the poster to exclude the QR code instructions, so that is the context of this change yes. I'll add verbiage at the top if that helps but we should always look to the Epic in which the ticket resides to understand it's context.

**Comment by kellysmith1008** [comment]
Going to read through the slack message again today and review Ben's component proposals and will start mocking up some ideas to run past folks by end of tomorrow or early Friday at the latest

**Comment by kellysmith1008** [comment]
@loripusey I'm not sure who, if anyone else, needs to validate the poster, but placing the ticket in validate just in case.

**Comment by loripusey** [comment]
@kellysmith1008 Did you put in slack for @stephenBarrs to review? Want to get his validation before I forward to Asheville & Richmond folks.
cc: @benbrasso-agile6

**Comment by kellysmith1008** [comment]
@loripusey He gave it the thumbs up in the slack thread, which I took to mean he's good with it - but I can reach out if you'd like as well!

**Comment by loripusey** [comment]
Nope, that's all we needed; I will let the Asheville & Richmond folks know that the final poster version is available (they already have the link)
cc: @stephenBarrs

**Comment by loripusey** [comment]
validated
